ctapipe/utils/astro.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.coordinates import Angle
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+import logging
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['get_bright_stars']
+
+
+def get_bright_stars(pointing=SkyCoord(ra=0. * u.rad, dec=0. * u.rad, frame='icrs'),
```

Comment:
I would not give a default for the pointing.

Reply:
corrected in 96c2b77

Reply:
No, now the default value is `None` which is not a valid input for `pointing`.

No default is different than `None` as default. Make t a positional argument, just `get_bright_stars(pointing, ...)`